### PR TITLE
use `useTheme` from nextui instead of next-themes

### PR DESCRIPTION
**Pull request by erenkulaksiz**
## [LEVEL]/[COMPONENT]
**TASK**: Potential fix for https://github.com/nextui-org/nextui/pull/200

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Only
- [ ] Refactor

### Description, Motivation and Context
Instead of getting isDark state from next-themes, it will be better to get it from next-ui to sync up with theme. Also this might fix the pull request i put above.

### CODE REVIEW

**Comment by jrgarciadev** [comment]
Hey @erenkulaksiz could you please also apply this to the `theme-switch` component? thanks 🙌🏼.

**Comment by erenkulaksiz** [comment]
Done 👍

**Comment by jrgarciadev** [comment]
Hey @erenkulaksiz  could you please resolve the comment?, you should remove theme `theme` object from  `const { setTheme, theme } = useNextTheme();` , before sending the commit please run `yarn build:docs`, thanks 🙏🏻

**Review by jrgarciadev — APPROVED** [review]

**Comment by erenkulaksiz** [comment]
> Hey @erenkulaksiz could you please resolve the comment?, you should remove theme `theme` object from `const { setTheme, theme } = useNextTheme();` , before sending the commit please run `yarn build:docs`, thanks 🙏🏻

Hey, sorry for the inconvenience. I fixed it and checked, https://github.com/nextui-org/nextui/pull/200 issue is now fixed.

File: apps/docs/src/components/theme-toggle/index.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ const StyledButton = styled('button', {
 export const ThemeToggle: React.FC<Props> = ({ className, css }) => {
   const isMounted = useIsMounted();
   const { setTheme, theme } = useNextTheme();
```
[the remark this answers is not in the document]
Comment (erenkulaksiz) [reply apps/docs/src/components/theme-toggle/index.tsx]: Please try pulling newest commit, i can build docs succesfully locally. Thanks 🙌

**Pull request merged**